example/eject.js

```diff
@@ -16,20 +17,20 @@
 
 'use strict';
 
-const mountutils = require('.');
-const drive = process.argv[2];
+const mountUtils = require('..');
 
-if (!drive) {
-  console.error(`Usage: ${process.argv[0]} ${process.argv[1]} <drive>`);
+let argv = process.argv.slice(2);
+let disk = argv.shift();
+
+if (!disk) {
+  console.error(`Usage: node scripts/eject ${process.argv[1]} <drive>`);
```

Comment:
Node doesn't require the `.js` extension to be specified, and `process.argv[1]` would be the full path to the source file being run. This is all a bit wrong, should be `node example/eject <drive>` – fixed now.